update/update-version.md

```diff
@@ -1,139 +1,187 @@
 # Updating from v3.x.x to v4.x.x
 
-Starting from version v4.x.x, we have refactored the code for the VSI module, resulting in a significant change when transitioning from version 3.x.x to 4.x.x. This means that the Virtual Server Instances (VSIs) managed by this module will be deleted and recreated during the update process.
+Version v4.x.x changes the VSI module code in ways that result in significant changes when you update from version 3.x.x to 4.x.x. When you update, the Virtual Server Instances (VSIs) that are managed by this module are deleted and re-created.
```

Comment:
I'm wondering if we should mention the type of change. From the end user perspective, the VSI naming convention is getting changes to this format: `prefix- + the last 4 digits of the subnet ID + a sequential number for each subnet`.

For example,
```sh
~ name                             = "prefix-001" -> "prefix-3ad7-001"
```

Reply:
Added this:

> VSIs in v4.x.x have a new prefix naming convention `prefix- + the last 4 digits of the subnet ID + a sequential number for each subnet`. For example, `prefix-3ad7-00`. When you update, your VSIs will adopt the new prefix.